backend/de.metas.business/src/main/java/de/metas/costrevaluation/CostRevaluationRepository.java

```diff
@@ -23,4 +38,89 @@ public List<I_M_CostRevaluationLine> retrieveLinesByCostRevaluationId(@NonNull f
 				.create()
 				.list();
 	}
+
+	public I_M_CostRevaluation getById(@NonNull final CostRevaluationId costRevaluationId)
+	{
+		return InterfaceWrapperHelper.load(costRevaluationId, I_M_CostRevaluation.class);
+	}
+
+	@NonNull
+	public CostRevaluation retrieveById(@NonNull final CostRevaluationId costRevaluationId)
```

Comment:
pls unify getById and retrieveById .... or make their name more meaningful.

> public I_M_CostRevaluation getById(@NonNull final CostRevaluationId costRevaluationId)

Check if you need it from outside. 

If not, pls inline it to `retrieveById` and rename `retrieveById` to `getById`.

In case it's needed then rename `getById` to `getRecordById`.
Also rename `retrieveById` to `getById`.